### DISCUSSION TITLE
Ability to filter nested entries by field values

### DISCUSSION THREAD

**Discussion by jamiepittock** [craftcms/cms#16289] [Ideas] [answered]
I'm not sure if this is an idea or a question, because it could be possible and I'm just missing something!

Regular section entries can be filtered by their field values:

![Screenshot 2024-12-06 at 14 20 41](https://github.com/user-attachments/assets/a5a75ec5-e763-487b-91f0-81bc98ab72f1)

Nested matrix entries don't seem to have that feature:

![Screenshot 2024-12-06 at 14 17 07](https://github.com/user-attachments/assets/bb1817a9-d454-4fa6-9e49-6358ce4f152f)

Is there a reason why, or am I missing a setting to allow it?

**Comment by brandonkelly** [comment, marked as answer]
This got slightly better in Craft 5.5.5 thanks to 3a34393634d39681105c738f0c270ab1857ab80a– now all custom fields selected by entry types’ field layouts are included in that menu.

Just made this better for 5.6+ though. Going forward, the menu will only include custom fields that are actually included in the nested entry types’ field layouts. (cf5c8b20f06c692b277e7d1d75b6123fa17931f3)

**Reply by brandonkelly** [reply]
Craft 5.6.0 is out with that additional change 🚀
